src/test/resources/docker-compose-test.yaml

```
@@ -4,17 +4,27 @@ services:
 
   #databases
   setup:
-    image: ghcr.io/gamify-it/test-data:main
+    image: ghcr.io/gamify-it/test-data:maintenance-keycloak-clone-test-data
```

```suggestion
    image: ghcr.io/gamify-it/test-data:main
```
Do you change this back?

---

can be changed back after https://github.com/Gamify-IT/test-data/pull/17 is merged